sirbot/plugins/slack/endpoints.py

```diff
@@ -21,7 +21,9 @@
         if slack.signing_secret:
             try:
                 validate_request_signature(
-                    await request.read(), request.headers, slack.signing_secret
+                    (await request.read()).decode("utf-8"),
```

Comment:
I nerve like this syntax around `await` with parenthesis. in find it more readable to do that in two steps.